server/__test__/AccountsService.test.js

```diff
@@ -0,0 +1,171 @@
+const {
+    createNewAccount,
+    accountDoesExist,
+    validateFields,
+    containsSpecialCharacters
+} = require('../src/service/AccountsService');
+const accountsDao = require('../src/repository/AccountsDAO');
+
+
+describe('createNewAccount Tests', () => {
+
+    test('should return data successfully', async () => {
+        const receivedData = {
+            username: 'test_username',
+            password: 'password123',
```

Comment:
You should use constants for these values and replace all instances of these values with the constants.

Reply:
I will keep this in mind going forward